src/main.py

```diff
@@ -54,14 +55,43 @@ def frontend_communication_upload_recorded():
         return jsonify({
             "error": "Expected file named recordingTemp"
         })
-    filename_ogg = app.config['UPLOAD_FOLDER'] / "recordingTemp.ogg"
+    filename_ogg = app.config['TEMP_FOLDER'] / "recordingTemp.ogg"
     file.save(filename_ogg)
-    filename_ogg = pathlib.Path(filename_ogg)
-    filename = app.config['UPLOAD_FOLDER'] / "recordingTemp.wav"
-    convert_recorded_file(filename_ogg, filename)
+    filename = app.config['TEMP_FOLDER'] / "recordingTemp.wav"
+    convert_recorded_to_wav(filename_ogg, filename)
     return jsonify({"filename": str(filename)})
 
 
+@app.route('/saveWithChords', methods=['POST'])
+def frontend_communication_save_with_chords():
+    try:
+        filename_dest = request.json["output_file"]
+    except Exception as e:
+        logger.error(f"Bad request: {request}\n Exception: {e}")
+        return jsonify({
+            "error": "Expected json with output_file key"
+        })
+    filename_src = app.config['TEMP_FOLDER'] / "output.midi"
+    music_synthesis.save_midifile_as_wav(filename_src, filename_dest)
```

Comment:
This line does not work on Windows, as it didn't work before

Reply:
Thats why it was only used as optional argument
```python
if args.wav:
        result_file = music_synthesis.save_midifile_as_wav("output.midi",
                                                           "output.wav")
```

Reply:
We agreed that output midi file is enough and we don't need to convert it to wav. I use it only as general utility in order to send files via facebook messanger (because it doesn't accept midi files)

Reply:
Then we would have to convert ogg file from recording to midi file :/ Also, have you noticed, that parselmouth cannot open the output.midi file? It says that it is not a music file. I think that's a clue that we might be doing sth wrong while saving this midi file

Reply:
Parselmouth cannot open the output.midi file because it is library that works on sound frequency, and "MIDI file does not need to capture and store actual sounds. Instead, the MIDI file can be just a list of events which describe the specific steps that a soundcard or other playback device must take to generate certain sounds"

Reply:
And midi2audio doesn't support windows, so this is the problem

Reply:
Ok, we should deal with this during the next week. However, I think this PR should be merged anyway